_source/logzio_collections/_metrics-sources/Prometheus-cloudwatch-exporter.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Ship Prometheus metrics from AWS CloudWatch with the OpenTelemetry Collector
+logo:
+  logofile: aws-cloudwatch.svg
+  orientation: vertical
+data-source: Amazon CloudWatch for Prometheus metrics
+templates: ["docker-metricbeat"]
+open-source:
+  - title: 
+    github-repo: 
+contributors:
+  - yotamloe
+  - yberlinger
+shipping-tags:
+  - aws
+---
+
+# logzio-aws-metrics
```

Comment:
@yotamloe  

Why is this tag here? Does it have a purpose?:  **# logzio-aws-metrics**

Reply:
That's the name of the GitHub project